python/housinginsights/sources/wmata_distcalc.py

```diff
@@ -228,6 +229,38 @@ def _get_walking_distance(self, srcLat, srcLon, destLat, destLon):
            destLon - longitude for destination location
            mapbox_api_key - api key for mapbox REST services
            """
+
+        if self.use_cached_distance == True:
+            try:
+                #Configure the connection
+                engine = dbtools.get_database_engine(db)
+                conn = dbtools.get_database_connection(db)
+                logging.info("  Connected to Housing Insights database")
+
+                #Pull columns to see if the database has updated columns in wmata_dist
+                columnset = conn.execute('select column_name from INFORMATION_SCHEMA.COLUMNS where TABLE_NAME=\'wmata_dist\'')
+                columns = []
+                for c in columnset:
+                    columns.append(c)
+
+                if ( 'building_lat' in columns and 'building_lon' in columns and 'stop_or_station_lat' in columns and 'stop_or_station_lon' in columns ):
+                #See if row exists
+                    proj_query = 'select * from wmata_dist where building_lat=\'' + srcLat + '\' and building_lon=\'' + srcLon + '\' and stop_or_station_lat=\''+ destLat + '\' and stop_or_station_lon=\'' + destLon + '\''
+                    row = conn.execute(proj_query)
+                    if ( row != None ):
+                        conn.close()
+                        engine.dispose()
+                        result = []
+                        for r in row:
+                            result.append(r)
+                        return result
```

Comment:
See my comment below, I recommend switching this so that only the distance is returned so that _get_walking_distance behaves the same way in both cases.